### ISSUE TITLE
Document examples of copying, and include the --format=id feature

### ISSUE THREAD

**Issue by AlanCoding** [ansible/tower-cli#329]
We could use more elaboration and examples around the copy functionality.

Also, I think the new `--format=id` will be very helpful for that. Example

```
tower-cli inventory copy $(tower-cli inventory get --name="Alan AWS" --format=id)
```

A page inside of the docs for copying, in general, would be nice.

**Comment by munichman2008** [comment]
I totally agree. Would be great to get more documentation on this. Additionally, and as a result of perhaps lack of documentation, I noticed the copy command ignores the override of the name (it still just adds the timestamp). I will log an issue for this.
